### PR TITLE
Edit user page added

### PR DESCRIPTION
**Pull request by CarlosPrattMedrano**

### CODE REVIEW

File: frontend/src/app/services/typicode.service.ts [hunk 1]
```diff
@@ -27,4 +26,9 @@ export class TypicodeService {
   getAllAlbums(): Observable<IAlbum[]> {
     return this.http.get<IAlbum[]>(this.resourceUrl('albums'));
   }
+
+  updateUser(user: IUser): Observable<any> {
+    console.log("aaaa")
```
Comment (jrodrigoav) [inline frontend/src/app/services/typicode.service.ts:31]: Remove console.log when submitting PRs

**Review by jrodrigoav — COMMENTED** [review]
Just one comment

File: frontend/src/app/services/typicode.service.ts (hunk 1 above)
Comment (CarlosPrattMedrano) [reply frontend/src/app/services/typicode.service.ts]: Sorry, the change is ready

**Review by jrodrigoav — APPROVED** [review]
Looks good

**Pull request merged**